src/app/components/schedule/calendar/cal-popup/cal-popup.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-cal-popup',
+  templateUrl: './cal-popup.component.html',
+  styleUrls: ['./cal-popup.component.scss']
+})
+export class CalPopupComponent {
+  @Input() date_time: string;
+  @Input() title: string;
+  @Input() time: string;
+  @Input() venue: string;
+  @Input() url: string;
+  @Input() img_title: string;
+  @Input() duration: string;
```

Comment:
why??? it should be done better, max 1 Input

Reply:
First of all, I used one Input with object, which had all of this properties. But when I tried to use it in component I have the [Object object] as the result. So, I decided to pass values like a 'string' and it solved my problem. But I understand that my solution is't  the best practice... What I did wrong? How I can solve this problem in another way?